index.html

```diff
@@ -0,0 +1,898 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Alegreya+Sans+SC+Open+Sans"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="src/css/reset.css" />
+    <link rel="stylesheet" href="src/css/main.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <div>
+        <a>News</a>
+        <a>Events ▾</a>
+        <a>Business ▾</a>
+        <a>AMas</a>
+        <a>Spaces</a>
+      </div>
+
+      <div>
+        <div>
```

Comment:
use nav in main menu instead of div.there should be only one nav.

Reply:
Ok thanks for you feedback cheers!